client/src/pages/Profile.js

```diff
@@ -426,12 +427,14 @@ const Profile = ({
             <PlaceholderIcon />
             {isSelf && (
               <>
-                <CreatePostDiv>{t("post.create")}</CreatePostDiv>
-                <CreatePostIcon
-                  id={GTM.user.profilePrefix + GTM.post.createPost}
-                  src={createPost}
-                  onClick={onToggleCreatePostDrawer}
-                />
+                <CreatePostDiv id={GTM.user.profilePrefix + GTM.post.createPost} onClick={onToggleCreatePostDrawer}>
+                  <CreatePostIcon
+                    id={GTM.user.profilePrefix + GTM.post.createPost}
+                    src={createPost}
+                    onClick={onToggleCreatePostDrawer}
```

Comment:
Why do we have the event listener and the GTM id on the parent and the child. It should be only on one element.